addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,234 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
     &.o_kanban_ungrouped {
         .o-justify-content(center);
+        padding: 12px;
+        min-height: 100%;
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
+            .o-flex(0, 1, auto);
+            width: 50%;
+            margin: 0;
+
+            &:first-child, &:nth-child(2) {
+                @media (max-width: @screen-xs-max) {
+                    margin-top: 10px;
+                }
+            }
+
             @media (min-width: @screen-sm-min) {
-                width: 45%;
+                width: percentage(1/3);
+            }
+
+            @media (min-width: @screen-md-min) {
+                width: percentage(1/4);
             }
         }
     }
 
     .o_kanban_record {
-        padding: 0;
+        padding: 1% 3% 3%;
+        box-shadow: none;
+        background: none;
+        border: none;
+        -webkit-font-smoothing: antialiased;
+
+        .o_browser_bar {
+          background: white;
+          border-bottom: 1px solid darken(@o-theme-kanban-gray, 6%);
+          .size(100%, 20px);
+
+          &:before {
+            content: "•••";
+            display: block;
+            padding-left: 10px;
+            color: darken(@o-theme-kanban-gray, 6%);
+            font: 12px/20px Courier, monospace;
+            letter-spacing: -1px;
+          }
+        }
+
+        &.o_theme_installed .o_theme_preview_top {
```

Comment:
@qsm-odoo , honestly I didn't received very detailed specs about this task, but seems like you should be able to "switch theme" in anytime, using both odoo and 3th party themes.
By this point of view it make sense to me to highlight the installed one